resources/startup.sh

```diff
@@ -3,92 +3,83 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-SONAR_PROPERTIESFILE=/opt/sonar/conf/sonar.properties
-
-# get variables for templates
-ADMINGROUP=$(doguctl config --global admin_group)
+# import util functions:
+# sql()
+# add_temporary_admin_user()
+# remove_temporary_admin_user functions()
+# wait_for_sonar_status_endpoint()
+# wait_for_sonar_to_get_up()
+source util.sh
+
+SONAR_PROPERTIES_FILE=/opt/sonar/conf/sonar.properties
+
+# get variables
+ADMIN_GROUP=$(doguctl config --global admin_group)
 FQDN=$(doguctl config --global fqdn)
 DOMAIN=$(doguctl config --global domain)
-MAIL_ADDRESS=$(doguctl config -d "sonar@${DOMAIN}" --global mail_address)
-# shellcheck disable=SC2034
-DATABASE_TYPE=postgresql
+MAIL_ADDRESS=$(doguctl config --default "sonar@${DOMAIN}" --global mail_address)
 DATABASE_IP=postgresql
```

Comment:
I think we dont need these DB variables anymore since we are now using the doguctl template command.

Reply:
Unfortunately not, these settings are not accepted by SonarQube if they are set in the sonar.properties file.